### PR TITLE
Adding file for London UK

### LINKED ISSUE #5: Hometown for irmbrady
## Add a file for London, UK

- [x] Create a branch
- [x] Add a file for London
- [x] Commit
- [x] Open a Pull Request
- [ ] Have a discussion?
- [ ] Merge

### PR DESCRIPTION
**Pull request by ghost**
This is the initial commit for the London UK file. Some information taken from Wikipedia. This file will use Markdown formatting.

This closes #5 

Sorry @hcanna, you are the only @ username I know... what do you think? :wink:

### CODE REVIEW

File: places/london-uk.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# London
+
+London is the capital and most populous city of England and the United Kingdom. Standing on the River Thames, London has been a major settlement for two millennia. It was founded by the Romans, who named it Londinium.
+
+## Places to Eat
+- St George's Inn
```
Comment (irmbrady) [inline places/london-uk.md]: Are there any other places to eat?

**Comment by hcanna** [comment]
@irmbrady ... looks good to :ship:

File: places/london-uk.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- Franco Manco
+- Meat Liquor
+
+## Things to do
+- Queue for the Tube
+- Stand on the right of the escalator
+- Visit the Space Museum
+- Cycle in Hyde Park
```
Comment (curekli) [inline places/london-uk.md]: Can I borrow your bike ? @irmbrady

Comment (irmbrady) [inline places/london-uk.md]: Sorry @curekli! I don't own a bike... you can always hire one close by! :bike:

**Pull request merged**